extensions/cloudfiles-extensions/pom.xml

```diff
@@ -0,0 +1,186 @@
+		<jclouds.version>1.9.1</jclouds.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>io.druid</groupId>
+			<artifactId>druid-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.metamx</groupId>
+			<artifactId>emitter</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.inject</groupId>
+			<artifactId>guice</artifactId>
+			<version>3.0</version><!--$NO-MVN-MAN-VER$-->
+		</dependency>
+		<dependency>
+			<groupId>com.google.inject.extensions</groupId>
+			<artifactId>guice-servlet</artifactId>
+			<version>3.0</version><!--$NO-MVN-MAN-VER$-->
+		</dependency>
+		<dependency>
+			<groupId>com.google.inject.extensions</groupId>
+			<artifactId>guice-multibindings</artifactId>
+			<version>3.0</version><!--$NO-MVN-MAN-VER$-->
+		</dependency>
+		<!-- jclouds dependencies -->
+		<dependency>
+			<groupId>org.apache.jclouds.driver</groupId>
+			<artifactId>jclouds-slf4j</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.driver</groupId>
+			<artifactId>jclouds-sshj</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<!-- Rackspace US dependencies -->
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-cloudservers-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-cloudfiles-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-cloudblockstorage-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-clouddatabases-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-cloudqueues-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-cloudloadbalancers-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-clouddns-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-cdn-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-cloudnetworks-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<!-- Rackspace UK dependencies -->
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
```

Comment:
((I don't know much about rackspace)) Do we have to include every region's implementations as part of the default config? How does Rackspace recommend distributing the impl for particular regions?